mmrazor/models/losses/mgd_loss.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+import torch
+import torch.nn as nn
+
+from mmrazor.registry import MODELS
+
+
+@MODELS.register_module()
+class MGDLoss(nn.Module):
```

Comment:
Add the test case of `MGDLoss` in [test_distillation_losses](https://github.com/open-mmlab/mmrazor/blob/dev-1.x/tests/test_models/test_losses/test_distillation_losses.py).
Add the docstring of `get_dis_loss`.